tools/tests/rangeslider/test.js

```diff
@@ -121,6 +137,109 @@ describe('Rangeslider component', () => {
         assert.equal(left, value);
     });
 
+    it('Should change value dynamically', async () => {
+        await loadRangeslider({});
+        const rangeSlider = document.querySelector('gameface-rangeslider');
+        const rangesliderHandle = document.querySelector('.guic-horizontal-rangeslider-handle');
+        rangeSlider.value = 70;
+        let left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 70);
+
+        rangeSlider.setAttribute('value', 20);
+        left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 20);
+
+        rangeSlider.value = 'invalid';
+        left = parseFloat(rangesliderHandle.style.left);
+        assert.equal(left, 0);

```

Comment:
This is a negative test case, it should be separate. Maybe `Should set value to min if changed value is invalid`

Reply:
Fixed